src/pharmpy/modeling/error.py

```diff
@@ -895,22 +896,31 @@ def set_time_varying_error_model(
 def set_power_on_ruv(
     model: Model,
     list_of_eps: Optional[Union[str, list]] = None,
+    dv: Union[sympy.Symbol, str, int, None] = None,
     lower_limit: Optional[float] = 0.01,
     ipred: Optional[Union[str, sympy.Symbol]] = None,
     zero_protection: bool = False,
 ):
-    """Applies a power effect to provided epsilons.
+    """Applies a power effect to provided epsilons. If a dependent variable
```

Comment:
Note that the first line in the docstring is the "short" description of the function. Extended description should come later. I'll merge anyway and this can be fixed later.